lib/ibc-vm-rs/Cargo.toml

```diff
@@ -9,6 +9,7 @@ version      = "0.1.0"
 enumorph                 = { workspace = true }
 frame-support-procedural = { workspace = true }
 hex                      = { workspace = true }
+ibc-events               = { version = "0.1.0", path = "../ibc-events" }
```

Comment:
why don't we add these to workspace?

Reply:
ended up fixing this in this PR to trigger garnix